Visualizer/plotgame.py

```diff
@@ -1,16 +1,19 @@
 from pipper import install_packages
-install_packages(['pygame', 'euclid3'])
+install_packages(['pygame', 'numpy'])
 from rectangle import Rectangle
 from modules.CAN_Navigation.module.Algorithms import AStar
 from modules.CAN_Navigation.module.Structure import grid_factory
-import euclid3 as vmath
 import pygame as pg
+import numpy as np
 
 
-GRID_SIZE = 50
+GRID_SIZE = 60
 
 
 def main():
```

Comment:
Add docs of what the function actually does, you're using a polling event that waits for input but can have certain behavior that you avoid with your continue statements.

Reply:
Added docs for main and the continue statement.